src/sample/Main.java

```diff
@@ -3,19 +3,14 @@
 import javafx.application.Application;
 import javafx.scene.Scene;
 import javafx.stage.Stage;
-import java.util.ArrayList;
-
 
 // needs an extensive review before moving forward
 public class Main extends Application {
     private static Game game;
     static int gridSizeForGame;
     public static View gameView;
 
-    // implements your default size if no arg is provided and makes a 5x5 grid
-    private final ArrayList<String> parameters = new ArrayList<>();
-
-    private Stage startGUI = new Stage();
+    private static Stage startGUI = new Stage();
```

Comment:
We should probably make this a new class and extract this out, it's starting to violate Single Responsibility.

Reply:
just making the stage or setting up the various types of stages in a new class?

Reply:
like I can abstract out the startGUI into a new class for sure easy peasy but then a class for Stages might take some thinking since we have at this point three different stages
Of course we could find what they have in common and the rest make methods that the Main or View would call to generate the specific one and that is fair.